src/themes/structural/modules/Button.ts

```diff
@@ -0,0 +1 @@
+export default {"button":"_button_312p3_1"};
```

Comment:
Same for these, they shouldn't be checked in...

Also, to get around the tslint issue, if you emit:

```typescript
/* tslint:disable:object-literal-key-quotes quotemark */
export default {"button":"_button_312p3_1"};

```

It should get through tslint.